libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -149,3 +149,79 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
                 )
                 for page in doc.pages
             ]
+
+
+class AmazonTextractPDFParser(BaseBlobParser):
+    """Sends PDF files to Amazon Textract and parses them to generate Documents.
+    For parsing multi-page PDFs, they have to reside on S3.
+    """
+
+    try:
+        import textractcaller as tc
+    except ImportError:
+        raise ModuleNotFoundError(
+            "Could not import amazon-textract-caller python package. "
+            "Please install it with `pip install amazon-textract-caller`."
+        )
+
+    def __init__(
+        self, textract_features: List[tc.Textract_Features] = [], client: Any = None
+    ) -> None:
+        """
+        :func `Textract_features <tc.Textract_Features>`
```

Comment:
nit: may be helpful to document the parameters explicitly here, don't think documentation will surface for users  (https://api.python.langchain.com/en/latest/api_reference.html)